sdk/program/src/borsh.rs

```diff
@@ -159,4 +192,21 @@ mod tests {
                 + get_packed_len::<TestEnum>()
         );
     }
+
+    #[test]
+    fn instance_packed_len() {
```

Comment:
Be nice to see a few more tests here, otherwise, besides this nit, I approve!

Reply:
No problem, any other cases in particular that you'd like to see tested?